lib/email_validator.rb

```diff
@@ -55,7 +55,7 @@ def regexp(options = {})
     protected
 
     def loose_regexp(options = {})
-      return /\A[^\s]+@[^\s]+\z/ if options[:domain].nil?
+      return /\A[^\s]+@[^\s]+\z/ if options[:domain].nil? && !options[:require_fqdn]
```

Comment:
`require_fqdn` is not even considered in loose mode, which very much intentionally only looks for `@` in the string (see https://github.com/K-and-R/email_validator#validation-philosophy). If you are wanting to have any additional validation, you need strict mode.